### PR TITLE
Add hall events

### PR DESCRIPTION
**Pull request by jieffyy**
Implement the model and command to create hall events.

### CODE REVIEW

File: src/main/java/seedu/address/model/event/Event.java [hunk 1]
```diff
@@ -1,36 +1,87 @@
...
+            return true;
+        }
+
+        if (!(other instanceof Event)) {
+            return false;
+        }
+
+        Event otherEvent = (Event) other;
+        return otherEvent.getName().equals(getName())
+                && otherEvent.getDescription().equals(getDescription());
```
Comment (jieffyy) [inline src/main/java/seedu/address/model/event/Event.java:71]: Might need change this implementation to also compare the attendees list

File: src/main/java/seedu/address/model/event/Event.java [hunk 2]
```diff
@@ -1,36 +1,87 @@
...
+    }
+
     @Override
     public int hashCode() {
-        return eventName.hashCode();
+        return Objects.hash(eventName, description, attendeesList);
```
Comment (jieffyy) [inline src/main/java/seedu/address/model/event/Event.java:76]: Similarly, not sure if this hash will take into account 2 events with same name, description, but different attendeesList (or that it doesn't matter).

**Review by yanchenglee98 — APPROVED** [review]
LGTM

**Comment by jieffyy** [comment]
EDIT: was part of #91.

**Pull request merged**